sparkles/tests/test_yoshi.py

```diff
@@ -148,7 +148,7 @@ def test_get_params_use_cycle():
 
 
 @pytest.mark.skipif(not HAS_WEB_SERVICES, reason="No web services available")
-def test_acar_from_ocat(monkeypatch):
+def test_acar_from_ocat(proseco_agasc_1p7, monkeypatch):
```

Comment:
I note that this test probably doesn't need to use monkeypatch and explicitly disable the supplement (that should already be happening via autouse)-- but that's not an issue with this PR.